models/AlexNet.py

```diff
@@ -74,10 +56,10 @@
 model.summary()
 
 
-history = model.fit(x_train1,y_train1,batch_size = 32,epochs = 20,validation_data = (x_val,y_val))
+history = model.fit(cifar10.x_train1,cifar10.y_train1,batch_size = 32,epochs = 20,validation_data = (cifar10.x_val,cifar10.y_val))

```

Comment:
`x_train1` and `y_train1` sounds wrong, maintain better variable names within the class.